packages/bloc/test/cubit_test.dart

```diff
@@ -262,14 +275,15 @@ void main() {
 
       setUp(() {
         observer = MockBlocObserver();
-        Bloc.observer = observer;
       });
 
       test('triggers onClose on observer', () async {
-        final cubit = CounterCubit();
-        await cubit.close();
-        // ignore: invalid_use_of_protected_member
-        verify(() => observer.onClose(cubit)).called(1);
+        await BlocOverrides.runZoned(() async {
```

Comment:
Should we have some error handling for this async call?

Reply:
It’s not necessary since this is a year and an unhandled exception will result in a test failure